completions/chflags

```diff
@@ -0,0 +1,31 @@
+# chflags(1) completion                                      -*- shell-script -*-
+
+_comp_cmd_chflags()
+{
+    local cur prev words cword was_split comp_args
```

Comment:
```suggestion
    local cur prev words cword comp_args
```

`was_split` is only needed when the flag `-s` is specified to `_comp_initialize`.